### PR TITLE
(doc) Apply developmental edits by @kayayarai to static catalog docs.

### PR DESCRIPTION
**Pull request by gguillotte**
Resolves DOC-2593.

### CODE REVIEW

File: documentation/config_file_puppetserver.md [hunk 1]
```diff
@@ -32,36 +33,25 @@ The `puppetserver.conf` file contains settings for Puppet Server software. For a
...
+    * `environment-class-cache-enabled`: Optional. Used to control whether the master service maintains a cache in conjunction with the use of the [`environment_classes` API](./puppet-api/v3/environment_classes.html). If this setting is set to `true`, Puppet Server maintains the cache, enabling an Etag header to be returned for each GET request to the API and for subsequent GET requests using the prior Etag value in an If-None-Match header to return an HTTP 304 (Not Modified) response with no body when the class information available for an environment has not changed. If this setting is set to `false` or is not specified, Puppet Server doesn't maintain a cache, an Etag header is not returned for GET requests, and the If-None-Match header for an incoming request is ignored. Puppet Server therefore parses the latest available code for an environment from disk on every incoming request. For more information, see the [`environment_classes` API documentation][].
```
Comment (fpringvaldsen) [inline documentation/config_file_puppetserver.md]: > If this setting is set to `true`, Puppet Server maintains the cache...

This sentence is really long and I'm finding it hard to follow, might be worth breaking it up into two or three sentences?

File: documentation/config_file_puppetserver.md [hunk 2]
```diff
@@ -32,36 +33,33 @@ The `puppetserver.conf` file contains settings for Puppet Server software. For a
...
+
+    > **Note:** The `puppet-admin` setting and `client-whitelist` parameter are deprecated in favor of authorization methods introduced in Puppet Server 2.2. For details, see the [`auth.conf` documentation][].
     * `authorization-required` determines whether a client certificate is required to access the endpoints in this API. If set to `false`, all requests will be permitted to access this API. If set to `true`, only the clients whose certnames are included in the `client-whitelist` setting are allowed access to the admin API. If this setting is not specified but the `client-whitelist` setting is specified, the default value for this setting is `true`.
     * `client-whitelist` contains an array of client certificate names that are allowed to access the admin API. Puppet Server denies any requests made to this endpoint that do not present a valid client certificate mentioned in this array.
 
         If neither the `authorization-required` nor the `client-whitelist` settings are specified, Puppet Server uses the new authorization methods and [`auth.conf` documentation][] formats to access the admin API endpoints.
```
Comment (rlinehan) [inline documentation/config_file_puppetserver.md]: This isn't touched in this PR, but `the new authorization methods and auth.conf documentation formats` reads kind of odd... should that be `the new authorization methods and auth.conf format` ?

File: documentation/puppet-api/v3/static_file_content.md [hunk 3]
```diff
@@ -16,9 +16,9 @@ The `static_file_content` endpoint returns the standard output of a
 of a [file resource][] that has a `source` attribute with a `puppet:///` URI value. That
 source must be a file from the `files` directory of a module in a specific [environment][].
 
-Puppet Agent uses this endpoint only when applying a [static catalog][], and this endpoint
-is available only when the Puppet master is running Puppet Server. This endpoint does not
-exist on Ruby Puppet masters, such as the
+Puppet Agent uses this endpoint only when applying a [static catalog][]. This endpoint
+is available only when the Puppet master is running Puppet Server{~~. This endpoint does not
+exist on~>not ~~} Ruby Puppet masters, such as the
```
Comment (rlinehan) [inline documentation/puppet-api/v3/static_file_content.md]: Do these tildes do something in the docs on the website, or is this a typo?

Comment (gguillotte) [inline documentation/puppet-api/v3/static_file_content.md]: Nope, some feedback from Kari that slipped through BBEdit. :/

**Comment by rlinehan** [comment]
:+1:
